pallene/coder.lua

```diff
@@ -1408,7 +1405,7 @@ gen_cmd["CallStatic"] = function(self, cmd, func)
     local top = self:stack_top_at(func, cmd)
 
     local parts = {}
-    table.insert(parts, self:call_pallene_function(dsts, cmd.f_id, top, xs))
+    table.insert(parts, self:call_pallene_function(dsts, false, cmd.f_id, top, xs))
```

Comment:
I don't understand why the callstaic always says the function has no upvalues.

Reply:
Ah, I didn't request a review because I was still experimenting.
(If I did request a review by mistake, sorry about that, I also haven't fixed the eager upvalue fetching yet)

Reply:
Don't worry. Sometimes I think you requested a review even if you haven't. Those github notifications all look the same 🤷